src/components/SwapWidget/SelectTokenDrawer/index.tsx

```diff
@@ -64,7 +64,9 @@ const SelectTokenDrawer: React.FC<Props> = (props) => {
 
   const filteredTokens: Token[] = useMemo(() => {
     if (isAddressSearch) return searchToken ? [searchToken] : [];
-    return filterTokens(Object.values(allTokens), searchQuery);
+    const tokens = Object.values(allTokens);
+    tokens.unshift(CAVAX[chainId]);
+    return filterTokens(tokens, searchQuery);
```

Comment:
AVAX is removed from top position even if we don't search anything. 

![Screenshot 2022-06-01 at 3 32 00 PM](https://user-images.githubusercontent.com/6604146/171380298-144bd61e-e424-43de-8a5b-d40d996477ec.png)

it should work how it is working in prod.


https://user-images.githubusercontent.com/6604146/171380252-53699c85-8652-4010-999d-cd7f3f4b7eaf.mov